src/controllers/userController.js

```diff
@@ -96,16 +96,16 @@ export const forgotPassword = async (req, res) => {
 
     //send token too a user email
 
-    const resetURL = `${req.protocol}://${req.get('host')}/process.env.BASE_URL/resetpassword/${
+    const resetURL = `${process.env.BASE_URL}/reset-password?id=${
       user.id
-    }$/{user.passwordResetToken}`;
-
+    }&token=${resetToken }`;
```

Comment:
@uwishema11  why do you need the user id ?
can't it be in resetToken?

Reply:
I used it because  , the password will be updated to the user where Id  is that use Id in database

Reply:
@uwishema11  is this the url that will be redirected to at the frontend?

Reply:
If so the userId doesn't have to be publicly visible in the url.
It should be hashed  in the token

Reply:
I am storing token in redis, so we do not have  its column in postgresDb so that password can be updated where token is that resetToken. that is why  I decided to use the UserId so that the password will be updated  in database where Id is UserID